tests/functional/WebDriverWindowTest.php

```diff
@@ -48,6 +48,20 @@ public function testShouldMaximizeWindow()
         $this->assertGreaterThanOrEqual($sizeBefore->getHeight(), $sizeAfter->getHeight());
     }
 
+    /**
+     * @group exclude-chrome
+     */
+    public function testShouldMinimizeWindow()
+    {
+        $this->assertSame('visible', $this->driver->executeScript('return document.visibilityState;'));
```

Comment:
The test should also by skipped in JsonWire mode. This will also make it being skipped in HtmlUnit, as it is not W3C compatible afaik.

Reply:
Also added a JsonWire-only test to assert the fail.